### ISSUE TITLE
Tiptac blocked my UI 

### ISSUE THREAD

**Issue by Milky063** [frozn/TipTac#167]
**Describe the bug**
So i was starting off in a key and this error pop up in the time M+ run...

**To Reproduce**

**Expected behavior**

**Screenshots**

https://ibb.co/3YdMLGj

[https://ibb.co/3YdMLGj](url)

**TipTac Reborn (please complete the following information):**
 - Version 22.12.15
 

**WoW (please complete the following information):**
 - Flavor: retail
 - Version 10.0.2

**Additional context**

**Comment by StefGraf** [comment]
```
[ADDON_ACTION_BLOCKED] AddOn 'TipTac' tried to call the protected function 'MultiBarBottomLeftButton4:SetAttribute()'.
[string "@!BugGrabber/BugGrabber.lua"]:480: in function <!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `SetAttribute'
[string "@FrameXML/ActionButton.lua"]:350: in function `UpdatePressAndHoldAction'
[string "@FrameXML/ActionButton.lua"]:412: in function <FrameXML/ActionButton.lua:373>
[string "=[C]"]: in function `Update'
[string "@FrameXML/ActionButton.lua"]:362: in function `UpdateAction'
[string "@FrameXML/ActionButton.lua"]:821: in function `OnEvent'
[string "@FrameXML/ActionButton.lua"]:216: in function <FrameXML/ActionButton.lua:213>
```





```
[ADDON_ACTION_FORBIDDEN] AddOn 'TipTac' tried to call the protected function 'UseAction()'.
[string "@!BugGrabber/BugGrabber.lua"]:480: in function <!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `UseAction'
[string "@FrameXML/SecureTemplates.lua"]:364: in function `handler'
[string "@FrameXML/SecureTemplates.lua"]:690: in function <FrameXML/SecureTemplates.lua:672>
[string "@FrameXML/SecureTemplates.lua"]:704: in function <FrameXML/SecureTemplates.lua:697>
[string "@FrameXML/SecureTemplates.lua"]:746: in function `SecureActionButton_OnClick'
[string "@FrameXML/ActionButton.lua"]:109: in function `TryUseActionButton'
[string "@FrameXML/ActionButton.lua"]:144: in function `ActionButtonDown'
[string "ACTIONBUTTON2"]:2: in function <[string "ACTIONBUTTON2"]:1>
```

**Comment by fgmenth** [comment]
Did you guys happen to open your Talents window perhaps? This issue is 100% reproducible for me when I try to change my spec or talents.

**Comment by AevusErus** [comment]
Yes, I seem to encounter this issue after changing talents. I can't always get it to happen when changing specs or when changing from one preset build to another, but if I go in and start moving talent points around and save the changes, I pretty much always get this same error.

**Comment by fgmenth** [comment]
Then in this case this issue is a duplicate of #149

**Comment by frozn** [comment]
Should be fixed with latest release [v23.01.23](https://github.com/frozn/TipTac/releases/tag/v23.01.23-release). Please try again with this version.
